packages/client-sdk-nodejs/Makefile

```diff
@@ -28,3 +28,11 @@ test-topics-service:
 test-http-service:
 	@echo "Testing http service..."
 	@npm run integration-test-http
+
+test-full-network-outage-retry-service:
+	@echo "Testing full network outage retry service..."
+	@npm run integration-test-retry-full-network-outage
+
+test-temporary-network-outage-retry-service:
+	@echo "Testing temporary network outage retry service..."
+	@npm run integration-test-retry-temporary-network-outage
```

Comment:
Do we need separate targets for these? Also the `-service` suffix is to distinguish services in the canaries, eg leaderboard service vs cache vs topics. You can omit that unless this is something we're going to carry over to the canaries.